### PR TITLE
Lono templating

### PR DESCRIPTION
**Pull request by danielrbradley**
Generate variations for number of subnet groups and if the setup is publicly available.

### CODE REVIEW

File: .ruby-version [hunk 1]
```diff
@@ -0,0 +1 @@
+2.0.0
```
Comment (ValkyrieUK) [inline .ruby-version]: Lets bump this up to `ruby-2.4.1`. Ruby 2 is no longer maintained and `ruby_dep` needs at least 2.2.5 to install.

File: config/lono.rb [hunk 2]
```diff
@@ -0,0 +1,53 @@
+template "sentry-formation-public-1az.yaml" do
```
Comment (ValkyrieUK) [inline config/lono.rb]: Can probably do something like this

```ruby
3.times do |i|
  template "sentry-formation-public#{i + 1}az.yaml" do
    source "sentry-formation.yaml.erb"
    variables(
      :Description => "Sentry.io internet facing setup in only #{i + 1} availability zone",
      :visibility => "internet-facing",
      :availability_zones => i+1,
    )
  end
end

3.times do |i|
  template "sentry-formation-internal-#{i + 1}az.yaml" do
    source "sentry-formation.yaml.erb"
    variables(
      :Description => "Sentry.io internal setup in only #{i + 1} availability zone",
      :visibility => "internal",
      :availability_zones => i + 1,
    )
  end
end
```

**Comment by ValkyrieUK** [comment]
Also needed to install `brew cask install boltopslabs/software/bolts` as a dependancy for lono so probably worth updating the README

**Comment by ValkyrieUK** [comment]
Should the ELB have `0.0.0.0/0` for internal ? https://github.com/o2Labs/sentry-formation/blob/235efe173cdde1ca5d99af31fd70db011dddba64/templates/sentry-formation.yaml.erb#L656

File: config/lono.rb (hunk 2 above)
Comment (otaiga) [reply config/lono.rb]: or even `(1..3).each do |zones|` ;)

**Pull request merged**